Realm/Realm/Realm.csproj

```diff
@@ -10,7 +10,7 @@
   </PropertyGroup>
   <ItemGroup>
     <PackageReference Include="Newtonsoft.Json" Version="12.0.1" />
-    <PackageReference Include="Fody" Version="6.0.0" PrivateAssets="None" />
+    <PackageReference Include="Fody" Version="6.*" PrivateAssets="None" />
```

Comment:
shouldnt this be `PrivateAssets="All"` as well?

Reply:
We've always had Fody as an explicit dependency on the Realm SDK itself as it's impossible to use the SDK without Fody to run the Realm weaver.

Reply:
ahhh. makes sense. sorry for the noise

Reply:
Not at all - thank you for the input.